### ISSUE TITLE
kpi_without_optim vs target for Geo data

### ISSUE THREAD

**Issue by virithavanama** [google/lightweight_mmm#71]
Hi Team,
when i ran the find_optimal_budgets without geo data, the kpi_without_optim(14k) is closer to the original conversions/target(11.5k).
But the same data with geo is giving me weird values of 62million which are far off from the original 11k.
This is for 52 weeks of data but its the same case when I ran for the entire 195 weeks of data.  Do we need to rescale or something?
let me know if anything else is required from my end.

**Comment by pabloduque0** [comment]
No, `kpi_without_optim` should behave similarly between geo and national cases. 

Some of the inputs provided would had to be different at some point for those two to provide such different outputs.

**Comment by virithavanama** [comment]
no I tried complete data with extra features and just last 52 weeks and extra features. You can see from below that all other variables are same
# Run optimization with the parameters of choice.
solution, kpi_without_optim, previous_budget_allocation = optimize_media.find_optimal_budgets(
    n_time_periods=n_time_periods,
    media_mix_model=mmm,
    extra_features=extra_features,
    budget=budget,
    prices=prices,
    bounds_lower_pct=0.3,
    bounds_upper_pct=jnp.array([1,0.3580]),
    media_scaler=media_scaler,
    target_scaler=target_scaler,
    seed=SEED)

**Comment by pabloduque0** [comment]
I would need a reproducible example (can be a colab with mock data) in order to investigate further, its too broad many things play in on this one.

**Comment by virithavanama** [comment]
please give me an email id so that I can share my colab notebook

**Comment by pabloduque0** [comment]
Im sorry but that is not something we can do. If the error is reproducible, feel free to provide any mock data and code to reproduce and we can look into it.
